sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/CategorizedEntity.java

```diff
@@ -124,7 +126,7 @@ private void setLength(int length) {
      * @return the resolutions value.
      */
     public IterableStream<? extends BaseResolution> getResolutions() {
-        return this.resolutions;
+        return this.resolutions == null ? IterableStream.of(Collections.emptyList()) : resolutions;
```

Comment:
According to [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/IterableStream.java#L133), just use null as the argument.

```suggestion
        return this.resolutions == null ? IterableStream.of(null) : resolutions;
```